packages/expo-ui/ios/DisclosureGroupView.swift

```diff
@@ -0,0 +1,23 @@
+// Copyright 2025-present 650 Industries. All rights reserved.
+
+import ExpoModulesCore
+import SwiftUI
+
+class DisclosureGroupProps: ExpoSwiftUI.ViewProps {
+  @Field var title: String?
+  var onStateChange = EventDispatcher()
```

Comment:
if we add an event listener let's also add a`isOpen` prop and make this a controlled component.